src/main/java/appeng/tile/networking/TileEnergyAcceptor.java

```diff
@@ -84,11 +85,8 @@ protected double funnelPowerIntoStorage( final double power, final Actionable mo
 		{
 			final IEnergyGrid grid = this.getProxy().getEnergy();
 			final double leftOver = grid.injectPower( power, mode );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "leftOver". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1488)